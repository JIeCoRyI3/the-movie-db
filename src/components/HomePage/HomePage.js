import React from 'react';
import Header from '../Header/';
import MainContainer from '../MainContainer';
import SearchBar from '../Header/Mocks/MockSearchBar';

function HomePage() {
    return (
        <>
            <Header>
                <SearchBar />
            </Header>
            <MainContainer />
        </>
    );
}

export { HomePage };
