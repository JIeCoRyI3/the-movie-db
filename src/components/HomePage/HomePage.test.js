import React from 'react';
import { shallow } from 'enzyme';
import HomePage from './index';

describe('<Footer />', () => {
    const wrapper = shallow(<HomePage />);

    it('should be rendered', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
