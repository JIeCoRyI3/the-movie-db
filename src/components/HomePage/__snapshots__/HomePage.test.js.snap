// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> should be rendered 1`] = `
<Fragment>
  <withRouter(Connect(Header))>
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/film/:id"
      />
    </Switch>
  </withRouter(Connect(Header))>
</Fragment>
`;
